packages/geo/src/coordinates/latlon/internal/index.ts

```diff
@@ -0,0 +1,141 @@
+// __private-exports
+/*
+ * Copyright 2024 Hypergiant Galactic Systems Inc. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { Patterning } from '@/patterning';
+
+export type Errors = string[];
+
+export type Lat = 'LAT';
+export type Lon = 'LON';
+export type Format = `${Lat}${Lon}` | `${Lon}${Lat}`;
+
+export enum Compass {
+  North = 'N',
+  South = 'S',
+  East = 'E',
+  West = 'W',
+}
+
+export enum Limits {
+  Lat = 90,
+  Lon = 180,
+}
+
+export enum Symbols {
+  Degrees = '°',
+  Minutes = "'",
+  Seconds = '"',
+  Separator = '/',
+}
+
+export const BEARINGS = {
+  // [positive, negative] so that "isNegative" can be used to index into the array
+  LAT: [Compass.North, Compass.South],
+  LON: [Compass.East, Compass.West],
+} as const;
+
+export const DEFAULT_FORMAT: Format = 'LONLAT';
```

Comment:
Regarding a default ordering, we actually should do `LATLON` as that is what operators and such are use to (even though it is `LONLAT` in some specs like geojson).

Reply:
Personally, I would say "all" as it correlates to `x, y` /pedantic. My argument to that is, I would rather be correct as a whole than to change that correctness for in internal use case.